source/renderer/app/components/widgets/forms/PinCode.js

```diff
@@ -102,14 +131,15 @@ export default class PinCode extends Component<Props> {
               className={pinCodeClasses}
               label={null}
               key={key}
-              themeId={IDENTIFIERS.INPUT}
               skin={InputSkin}
               onChange={(number) => this.onChange(number, key)}
+              onKeyPress={(event) => this.onKeyPress(event, key)}
               value={value ? value[key] : undefined}
-              autoFocus={autoFocus && key === 0}
+              autoFocus={autoFocus}
+              allowSigns={false}
               disabled={
                 disabled ||
-                (key !== 0 &&
+                (key &&
                   (!value ||
                     !Object.prototype.hasOwnProperty.call(value, key - 1)))
```

Comment:
Please check if you can replace `!Object.prototype.hasOwnProperty.call(value, key - 1)))` by `!value[key]`